chaste_codegen/chaste_model.py

```diff
@@ -400,6 +402,16 @@ def _convert_membrane_stimulus_current(self):
         return membrane_stimulus_current_converted, \
             (self._membrane_stimulus_current_orig.units, membrane_stimulus_current_converted.units)
 
+    def _convert_other_currents(self):
+        """ Try to convert other currents (apart from stimulus current) to uA_per_cm2 """
+        currents = get_variables_transitively(self._model, (self._OXMETA, 'CellMembraneCurrent'))
+        currents = set(currents) - set(self._stimulus_params + [self._membrane_stimulus_current_converted])
+        for current in currents:
+            try:
+                self._model.convert_variable(current, self.units.get_unit('uA_per_cm2'), DataDirectionFlow.INPUT)
```

Comment:
```suggestion
                self._model.convert_variable(current, self.units.get_unit('uA_per_cm2'), DataDirectionFlow.OUTPUT)
```
Since Chaste is just reading currents, not modifying them - they're derived quantities, not parameters.